src/viking.js

```diff
@@ -1,11 +1,162 @@
 // Soldier
-class Soldier {}
+class Soldier {
+
+    constructor (health,strength) {
+    this.health = health;
+    this.strength = strength
+}
+//metodos soldier
+attack() {
+     
+    return this.strength
+};
+
+receiveDamage(damage) {
+
+    this.health = this.health - damage
+}
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+
+    constructor (name,health , strength) {
+      super (name,health , strength)
+
+    this.name = name
+    this.health = health
+    this.strength = strength }
+
+    //metodos viking
+
+    receiveDamage(damage){
+        
+        this.health = this.health - damage
+        let isStillAlive = `${this.name} has received ${damage} points of damage`
+        let vikingDies= `${this.name} has died in act of combat`
+        if ( this.health > 0 ) {
+            return isStillAlive
+        } else {
+            return vikingDies
+        }
+    }
```

Comment:
Esto esas variables las vuelves a reasignar? entonces siempre siempre siempre mejor const, vamos practicamente siempre sera const! :D
```suggestion
    receiveDamage(damage){
        this.health = this.health - damage
        const isStillAlive = `${this.name} has received ${damage} points of damage`
        const vikingDies= `${this.name} has died in act of combat`
        if ( this.health > 0 ) {
            return isStillAlive
        } else {
            return vikingDies
        }
    }
```